Cargo.toml

```diff
@@ -28,6 +28,7 @@ log = "0.4"
 bitflags = "1.3.2"
 reorder = "2.1"
 resources = "1.1"
+wasm-timer = "0.2"
```

Comment:
Wait, all you have to do is just add the dependency here? That's pretty neat. 

Can we add it only if the target is WASM? You can do that with something like:
```toml
[target.'cfg(target_arch = "wasm32")'.dependencies]
wasm-timer = "0.2"
```